tests/epyccel/test_epyccel_lists.py

```diff
@@ -26,6 +27,70 @@ def pop_last_element():
     assert isinstance(python_result, type(pyccel_result))
     assert python_result == pyccel_result
 
+def test_pop_list_bool(language) :
+    def pop_last_element():
+        a = [True, False, True]
+        return a.pop()
+    epyc_last_element = epyccel(pop_last_element, language = language)
+    pyccel_result = epyc_last_element()
+    python_result = pop_last_element()
+    assert isinstance(python_result, type(pyccel_result))
+    assert python_result == pyccel_result
+
+def test_pop_list_float(language) :
+    def pop_last_element():
+        a = [1.5 , 3.1, 4.5]
+        return a.pop()
+    epyc_last_element = epyccel(pop_last_element, language = language)
+    pyccel_result = epyc_last_element()
+    python_result = pop_last_element()
+    assert isinstance(python_result, type(pyccel_result))
+    assert python_result == pyccel_result
+
+def test_pop_list_of_lists(language) :
+    def pop_last_element():
+        a = [[4.6, 3.3], [4.2, 9.1], [2.3, 6.8]]
+        return a.pop()
+    epyc_last_element = epyccel(pop_last_element, language = language)
+    pyccel_result = epyc_last_element()
+    python_result = pop_last_element()
+    assert isinstance(python_result, type(pyccel_result))
+    assert python_result == pyccel_result
+
+def test_pop_list_of_tuples(language) :
+    def pop_last_element():
+        a = [(4.6, 3.3), (4.2, 9.1), (2.3, 6.8)]
+        return a.pop()
+    epyc_last_element = epyccel(pop_last_element, language = language)
+    pyccel_result = epyc_last_element()
+    python_result = pop_last_element()
+    assert isinstance(python_result, type(pyccel_result))
+    assert python_result == pyccel_result
+
+def test_pop_list_of_ndarrays(language) :
+    def pop_last_element():
+        import numpy as np
+
+        array1 = np.array([[1, 2, 3], [4, 5, 6]])
+        array2 = np.array([[7, 8, 9], [10, 11, 12]])
+        array3 = np.array([[13, 14, 15], [16, 17, 18]])
```

Comment:
I think you can work round the numpy reimport error by just importing what you need here:
```suggestion
        from numpy import array

        array1 = array([[1, 2, 3], [4, 5, 6]])
        array2 = array([[7, 8, 9], [10, 11, 12]])
        array3 = array([[13, 14, 15], [16, 17, 18]])
```

Reply:
i use it in another example,  will this work as well ?

Reply:
I think so. You'll have to run pylint to check but it's done in lots of other files